FLIR/conservator/wrappers/dataset_frame.py

```diff
@@ -96,3 +97,47 @@ def update_qa_status_note(self, qa_status_note: str):
         return self._conservator.query(
             Mutation.update_dataset_qa_status_note, input=note
         )
+
+    def add_dataset_annotations(self, dataset_annotation_create_list, fields=None):
+        """
+        Adds annotations using the specified list of `CreateDatasetAnnotationInput`
+        objects.
+
+        Returns a list of the added annotations, each with the specified
+        `fields`.
+        """
+        if dataset_annotation_create_list:
+
+            def ann_map_fn(annotation):
+                annotation.dataset_frame_id = self.id
+                return annotation
+
+            annotation_create_input = list(
+                map(ann_map_fn, dataset_annotation_create_list)
+            )
+
+            print(annotation_create_input)
```

Comment:
Was the print() statement meant for release?

Reply:
No, I put it in while testing. I'll remove it.